internal/codegen/imports.go

```diff
@@ -99,6 +99,7 @@ func importPathToAssumedName(importPath string) string {
 	}
 	base = strings.TrimPrefix(base, "go-")
 	if i := strings.IndexFunc(base, func(r rune) bool {
+		//nolint:QF1001 // De Morgan's law intentionally not applied
 		return !('a' <= r && r <= 'z' || 'A' <= r && r <= 'Z' || '0' <= r && r <= '9' || r == '_' ||
 			r >= utf8.RuneSelf && (unicode.IsLetter(r) || unicode.IsDigit(r)))
```

Comment:
I wonder why this ignore-comment won't work... 🤔 
I feel somewhat reluctant to changing the expression although it can be auto-fixed.

![Screenshot 2025-03-24 at 14 46 25](https://github.com/user-attachments/assets/8e639a20-0551-461c-988a-f684a01e4317)

Reply:
Ah, `//nolint:staticcheck` worked.